### PR TITLE
Support jerryscript on TizenRT

### PR DESCRIPTION
**Pull request by hs0225**
JSRemoteTest-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: API/device/artik053.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get_test_path(self):
         '''
         Return the test path on the device.
         '''
-        return '/rom/test'
+        return '/rom'
```
Comment (rtakacs) [inline API/device/artik053.py]: Was the path incorrect in the previous patch, right? As I checked all the tests are copied to a `res` folder that is mounted as `/rom`.

Comment (hs0225) [reply API/device/artik053.py]: No. The previous patch is correct. I just changed the copy location to match the test location of iot.js and jerryscript in the target.

Before
```
iot.js : /rom/test
jerryscript: /rom/jerry
```

After
```
iot.js : /rom
jerryscript: /rom
```

Comment (rtakacs) [reply API/device/artik053.py]: I see.

**Comment by rtakacs** [comment]
LGTM

**Comment by robertsipka** [comment]
LGTM

**Pull request merged**